src/main/java/address/MainApp.java

```diff
@@ -48,18 +51,19 @@ public void init() throws Exception {
 
     @Override
     public void start(Stage primaryStage) {
+        LoggerProvider.setLevel(Level.SEVERE);
         setupComponents();
         mainController.start(primaryStage);
 
-        EventManager.getInstance().post(new LoadDataRequestEvent(PreferencesManager.getInstance().getPersonFile()));
+
+        // initial load (requires mainController to be started.
+        EventManager.getInstance().post(new LoadDataRequestEvent(PrefsManager.getInstance().getSaveLocation()));
         syncManager.startSyncingData(config.updateInterval, config.simulateUnreliableNetwork);
     }
 
     protected void setupComponents() {
         config = getConfig();
-        PreferencesManager.setAppTitle(config.appTitle);
-
-        modelManager = new ModelManager(new AddressBook());
+        modelManager = new ModelManager(new AddressBook()); // empty model first
```

Comment:
Not sure if the comment is necessary, especially that `MainApp` does not need to know about it.